### ISSUE TITLE
New UI doesn't have space members

### ISSUE THREAD

**Issue by Sleuth56** [vector-im/element-web#20370]
### Steps to reproduce

1. Right click on a space.
2. See that there isn't a button to show space members.

This used to exist before the new UI changes.
It exists in version:
Element version: 1.9.7
Olm version: 3.2.8

#### What did you expect?
A button to show members of a space.
#### What happened instead?
There is no button when you right click on a space to show space members.

### Operating system

Manjaro

### Application version

Element Nightly version: 2021122001 Olm version: 3.2.8

### Will you send logs?

No

**Comment by Sleuth56** [comment]
A work around is, explore rooms in a space then click on the row of profile icons. I still think there should be a members button when someone right clicks on a space.

**Comment by SimonBrandner** [comment]
I believe this is an intentional change as the button was quite confusing to some

**Comment by Sleuth56** [comment]
I'm not sure how someone could find that confusing personally. I created this issue after I spent several minutes looking for the members panel to manage a space I'm the admin of.

**Comment by SimonBrandner** [comment]
See https://github.com/vector-im/element-web/issues/17376, https://github.com/vector-im/element-web/issues/16911, https://github.com/vector-im/element-web/issues/17018

**Comment by Sleuth56** [comment]
Those all look like functionality issues with the side panel not the UX being confusing.

**Comment by SimonBrandner** [comment]
> Those all look like functionality issues with the side panel not the UX being confusing.

The problem was that when showing the space member list, the member list you saw wasn't the member list of the room you were looking at. It then became unclear if the member list should change when switching rooms and what should happen when viewing a space home with a member list of another space shown.

That, at least from my perspective, qualifies as confusing :man_shrugging:  While I don't think the current behaviour is perfect, it at least doesn't feel confusing

**Comment by Sleuth56** [comment]
Yes, your right that is confusing. Guess UX has some work to do :) Maybe a general pannel for a space which shows all rooms and subspaces members and so on. Kinda a dashboard for the space.

**Comment by SimonBrandner** [comment]
> Maybe a general pannel for a space which shows all rooms and subspaces members and so on. Kinda a dashboard for the space.

We already have that :P The whole time, I sorta assume you know about that. If you right-click a space and click `Manage & explore rooms` you'll get that along with the member list

**Comment by Sleuth56** [comment]
See but there are two key things a user would have to know to access that. One they have to know to look in Explore rooms then they would have to know to click on the account icons to bring the members panel up. Maybe some better wording on the right click option but that's pretty good. My main complaint is there isn't an obvious button to open the members panel. I found it by a miss click in the UI while looking for it. I just don't think users would normally click on those.

**Comment by MadLittleMods** [comment]
For reference, this is how you view the space members currently:

 1. Go to the space home via the **Space home** option in the heading context menu
    - **Explore rooms** (or **Manage & explore rooms** if you're a space admin) go to the same place
 1. Click the **X members** text (this is very subtle)
 1. The member list should open on the right

1 | 2 | 3
--- | --- | ---
<img width="396" alt="" src="https://user-images.githubusercontent.com/558581/179869602-e90d93da-fddb-4cad-8cbd-b531343d22d5.png"> | <img width="357" alt="" src="https://user-images.githubusercontent.com/558581/179868980-c3958e58-77d9-4a94-b29f-5e601384787c.png"> | <img width="862" alt="" src="https://user-images.githubusercontent.com/558581/179868981-9519adf6-88f8-41b9-98dc-0020550a8962.png">
